code/modules/goonchat/browserOutput.dm

```diff
@@ -258,6 +258,12 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 		// url_encode it TWICE, this way any UTF-8 characters are able to be decoded by the Javascript.
 		C << output(url_encode(url_encode(message)), "browseroutput:output")
 
+/proc/to_chat(target, message, handle_whitespace = TRUE)
+	if(!SSchat)
+		to_chat_immediate(target, message, handle_whitespace)
```

Comment:
this path will never run unless someone finds a new sdql exploit to harddel the subsystem

Reply:
`if(Master.current_runlevel == RUNLEVEL_INIT || !SSchat?.initialized)` this is the current if check here, you should update the PR @oranges as there were some important QoL fixes with loading

Reply:
👌 , thank you